.changes/a7f82a33-11f1-4184-97af-ff713e922dfc.json

```diff
@@ -0,0 +1,9 @@
+{
+    "id": "a7f82a33-11f1-4184-97af-ff713e922dfc",
+    "type": "bugfix",
+    "description": "⚠️ **IMPORTANT**: Fix codegen for map shapes which use string enums as map keys. See the Map Enum Keys breaking change announcement (link coming soon) for more details",
```

Comment:
question: will this ship with the link or will it be added after?

Reply:
It will ship with the link. Now that the PR is approved, I'll remove the **BREAKING-CHANGE.tmp.md** file, post it as a GH discussion, update the changelog to include a link, and then merge the code to **main**.